docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -20,7 +20,7 @@ The following commands are examples. Before executing these example commands on
 
 :::
 
-1. Add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](use-configuration-file).
+1. Upgrade to Besu version 24.6.0 or add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](use-configuration-file).
```

Comment:
I think this is a preq and not part of the step

Reply:
Maybe just a note above with the following:

```
:::note
You are using Besu version 24.6.0 or later
::::
```

or 

```
Ensure you are using Besu version 24.6.0 or later
```

Reply:
With this one it's either/or: if you're on 24.6.0 it's enabled by default so you don't need to add the flag. If you're on an older version, you must use the experimental form of the option, --Xbonsai-limit-trie-logs-enabled

Reply:
Maybe it's fine to assume they are on 24.6.0+ if they're reading this version of the docs, in which case we could remove the step entirely.

Reply:
I've removed this step and added the note as a prerequsite [`5a93729` (#1627)](https://github.com/hyperledger/besu-docs/pull/1627/commits/5a9372992fa9ea60fb9592ea9f9e7b1a0ae384d2)